classes/spec_extensions.rb

```diff
@@ -85,6 +85,8 @@ def or_license_name_and_url
           return { license: "Apache 2", url: "https://www.apache.org/licenses/LICENSE-2.0.html" }
         elsif license.scan(/apache2/).count > 0
           return { license: "Apache 2", url: "https://www.apache.org/licenses/LICENSE-2.0.html" }
+        elsif license.scan(/apache 2/).count > 0
```

Comment:
these could be consolidated to `if license =~ /apache (license, version ) 2(\.0)?/i`